yarn.lock

```diff
@@ -3355,7 +3355,7 @@
   resolved "https://registry.yarnpkg.com/@jbx-protocol/contracts-v1/-/contracts-v1-2.0.0.tgz#616dd2adb8603dd1067d29fa613e704737adc243"
```

Comment:
think this diff was left over from when you installed the contract abis.

give it a `git checkout main -- yarn.lock`